quickwit/quickwit-indexing/src/source/kafka_source.rs

```diff
@@ -545,11 +534,6 @@ impl Source for KafkaSource {
         _ctx: &SourceContext,
     ) -> anyhow::Result<()> {
         self.poll_loop_jh.abort();
-
-        let consumer = self.consumer.clone();
-        spawn_blocking(move || {
-            consumer.unsubscribe();
```

Comment:
so unsubscribe is not required? Kafka does it on drop?

Reply:
I don't know, and the documentation is unclear. `close` is called on drop. `unsubscribe` removes the consumer from the group while leaving the connection opened, whereas `close` drops the connection.

We still call `unsubscribe` in the poll loop.